subsys/usb/class/netusb/CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ zephyr_library_include_directories(
   )
 
 zephyr_library_sources_ifdef(
+  CONFIG_USB_DEVICE_NETWORK
```

Comment:
This should be removed. This entire directory is ifdef'd on CONFIG_USB_DEVICE_NETWORK
one level up.

add_subdirectory_ifdef(CONFIG_USB_DEVICE_NETWORK netusb)

Reply:
So how can I include netusb.c?

Reply:
zephyr_library_sources(
  netusb.c
  )